src/Carnac.Tests/MessageProviderFacts.cs

```diff
@@ -61,12 +67,12 @@ public void recognises_shortcuts()
         }
 
         [Fact]
-        public void does_not_show_shortcut_name_on_partial_match()
+        public void does_not_show_key_press_on_partial_match()
```

Comment:
Not too many apps do chord style shortcuts. Also feels weird to show 'ctrl+r' which does nothing because the app is waiting for the second half. Hiding incomplete shortcuts feels right